src/spatialdata/_core/query/relational_query.py

```diff
@@ -10,6 +10,7 @@
 
 import dask.array as da
 import numpy as np
+import numpy.typing as npt
```

Comment:
I'm removing this in the next commit, using `from spatialdata._types import ArrayLike` instead.